pages/main.js

```diff
@@ -0,0 +1,16 @@
+import { StyledCard } from "../components/StyledCard";
+
+
+export default function CardList(){
+    return(
+        <ul>
+            <StyledCard>Modul Eins</StyledCard>
+            <StyledCard>Modul Zwei</StyledCard>
+            <StyledCard>Modul Drei</StyledCard>
+            <StyledCard>Modul Vier</StyledCard>
+            <StyledCard>Modul Fünf</StyledCard>
+            <StyledCard>Modul Sechs</StyledCard>
```

Comment:
you should consider making this part of the app dynamic at a later point and work with `.map()`